app/controllers/submit.js

```diff
@@ -246,7 +248,12 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
     }),
     titleChanged: Ember.computed('node.title', 'nodeTitle', function() {
         // Does the pending title differ from the title already saved?
-        return this.get('node.title') !== this.get('nodeTitle');
+        if (this.get('nodeTitle') != null) {
+            return this.get('node.title') !== this.get('nodeTitle').toString();
```

Comment:
Can you clarify when nodeTitle needs to be converted to a string? I thought it was bound to the value of a text box?